server/container/util/src/main/java/org/apache/james/util/CompletableFutureUtil.java

```diff
@@ -121,4 +122,10 @@
             return CompletableFuture.completedFuture(b);
         };
     }
+
+    public static <T> CompletableFuture<Stream<T>> sorted(CompletableFuture<Stream<T>> futureStream, Comparator<T> comparator) {
```

Comment:
I would not make that a CompletableFutureUtil API, it's very specific and unlikely to be used often